actions/verify-hello-app/action.yml

```diff
@@ -0,0 +1,68 @@
+name: 'Verify Hello App'
+description: 'Ensure a hello-app style script outputs the expected response'
+inputs:
+  command:
+    description: 'Command to run the hello-app'
+    required: true
+  use_server_key:
+    description: 'Populate LAUNCHDARKLY_SERVER_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+  use_client_key:
+    description: 'Populate LAUNCHDARKLY_CLIENT_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+  use_mobile_key:
+    description: 'Populate LAUNCHDARKLY_MOBILE_KEY with a server-side SDK if true'
+    required: false
+    default: 'false'
+
+
+runs:
+  using: composite
+
+  steps:
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_server_key == 'true' }}
+      name: 'Get the server SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/server-key = LAUNCHDARKLY_SERVER_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_client_key == 'true' }}
+      name: 'Get the client SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/client-key = LAUNCHDARKLY_CLIENT_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
+      if: ${{ inputs.use_mobile_key == 'true' }}
+      name: 'Get the mobile SDK key'
+      with:
+        aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+        ssm_parameter_pairs: '/sdk/common/hello-apps/mobile-key = LAUNCHDARKLY_MOBILE_KEY'
+
+    - uses: launchdarkly/gh-actions/actions/release-secrets
```

Comment:
I have the flag key as a secret since it makes it easier to change everywhere. Probably doesn't matter as much if we have a shared action, so we could inline it if we wanted. However, some SDKs will probably not be able to use this action (looking at you Roku), so there may be some duplication there.